src/theme/useDarkMode.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+export const useDarkMode = () => {
```

Comment:
it is not a useDarkMode thing. it is useThemeToggle thing. Feel the difference. What if the tell you to change green theme over red theme - which one is dark ?